Code/Framework/AzQtComponents/AzQtComponents/Gallery/GradientSliderPage.cpp

```diff
@@ -50,4 +50,9 @@ slider->updateGradient();
     ui->exampleText->setHtml(exampleText);
 }
 
+GradientSliderPage::~GradientSliderPage()
+{
+    delete ui;
+}
```

Comment:
If this was just empty, for example, the docs state this would still work with a scoped poitner.  It just can't be auto generated or in the header inlined.